apps/fluent-tester/src/FluentTester/TestComponents/Avatar/AvatarTest.tsx

```diff
@@ -1,128 +1,29 @@
-  const [showCustomBorderImage, setShowCustomBorderImage] = React.useState(false);
-  const [showRingGap, setShowRingGap] = React.useState(false);
-
-  return (
-    <View style={commonStyles.root}>
-      {/* settings */}
-      <View style={commonStyles.settings}>
-        <View style={commonStyles.switch}>
-          <Text>Custom foreground</Text>
-          <Switch value={showCustomForeground} onValueChange={setShowCustomForeground} />
-        </View>
-        <View style={commonStyles.switch}>
-          <Text>Custom background</Text>
-          <Switch value={showCustomBackground} onValueChange={setShowCustomBackground} />
-        </View>
-        <View style={commonStyles.switch}>
-          <Text>Custom ring color</Text>
-          <Switch value={showCustomRingColor} onValueChange={setShowCustomRingColor} />
-        </View>
-        <View style={commonStyles.switch}>
-          <Text>Custom Border Image</Text>
-          <Switch value={showCustomBorderImage} onValueChange={setShowCustomBorderImage} />
-        </View>
-        <View style={commonStyles.switch}>
-          <Text>Show Ring Gap</Text>
-          <Switch value={showRingGap} onValueChange={setShowRingGap} />
-        </View>
-      </View>
-      {/* component under test */}
-      <Avatar
-        primaryText="Kat Larrson"
-        secondaryText="Kat.Larrson@example.com"
-        ringColor={showCustomRingColor ? 'red' : null}
-        foregroundColor={showCustomForeground ? 'green' : null}
-        backgroundColor={showCustomBackground ? 'blue' : null}
-        customBorderImageSource={showCustomBorderImage ? rainbowGradientSource : null}
-        isRingVisible={true}
-        hasRingInnerGap={showRingGap}
-        size={'xxLarge'}
-      />
-    </View>
-  );
-};
-
-const AvatarSizeRamp: React.FunctionComponent = () => {
-  const allSizes: Size[] = ['xSmall', 'small', 'medium', 'large', 'xLarge', 'xxLarge'];
-
-  return (
-    <Stack style={{ flexDirection: 'row' }}>
-      {allSizes.map((size, index) => (
-        <Avatar key={index} primaryText="Kat Larrson" secondaryText="Kat.Larrson@example.com" size={size} />
-      ))}
-    </Stack>
-  );
-};
+import { StandardUsage } from './BasicAvatar';
+import { CustomizeUsage } from './CustomizedAvatar';
+import { E2EAvatarTest } from './E2EAvatarTest';
 
 const avatarSections: TestSection[] = [
   {
-    name: 'Basic Avatar',
-    testID: AVATAR_TESTPAGE,
-    component: BasicAvatar,
+    name: 'Standard Usage',
+    testID: JSAVATAR_TESTPAGE,
+    component: StandardUsage,
   },
   {
-    name: 'Custom Colors',
-    testID: AVATAR_TESTPAGE,
-    component: CustomizeColors,
+    name: 'Customize Usage',
+    component: CustomizeUsage,
   },
   {
-    name: 'Size Ramp',
-    component: AvatarSizeRamp,
+    name: 'Avatar E2E',
+    component: E2EAvatarTest,
   },
 ];
 
-export const AvatarTest: React.FunctionComponent = () => {
+export const JSAvatarTest: React.FunctionComponent = () => {
```

Comment:
Could we name this "AvatarTest", and name NativeAvatar, "NativeAvatarTest"?

Reply:
Thank you for noticing it :) I missed that part. Fixed